post_office/models.py

```diff
@@ -242,6 +242,7 @@ class Attachment(models.Model):
     name = models.CharField(_('Name'),max_length=255, help_text=_("The original filename"))
     emails = models.ManyToManyField(Email, related_name='attachments',
                                     verbose_name=_('Email addresses'))
+    mimetype = models.CharField(max_length=255, null=True)
```

Comment:
Could you please change this to `models.CharField(max_length=255, blank=True, default='')`?